src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,15 +1,29 @@
 package core.mate.academy.model;
 
+import java.util.List;
+
 /**
  * Add some custom fields that could be only in Bulldozer
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private double power;
+    private String producer;
+
     public Bulldozer() {
     }
 
     @Override
     public void doWork() {
         System.out.println("Bulldozer started to work");
     }
+
+    @Override
+    public List<Machine> get() {
```

Comment:
The `get` method is not typically a method you would override in a class like `Bulldozer`. This method seems to be creating instances of `Bulldozer` and returning them as a list, which is not a standard behavior for a class representing a single machine. Consider removing this method or ensuring it aligns with the intended design pattern.